tests/ui/privacy/associated-item-privacy-inherent.rs

```diff
@@ -1,5 +1,5 @@
 #![feature(decl_macro, associated_type_defaults)]
-#![allow(unused, private_in_public)]
+#![allow(unused)]
```

Comment:
`unused` is allowed by default in ui tests, whenever you see `allow(unused)` you can remove it as well.